docs/sources/configure/options.md

```diff
@@ -188,6 +188,17 @@ formats are:
 | `json`        | prints a compact JSON object   |
 | `json_indent` | prints an indented JSON object |
 
+| YAML               | Environment variable     | Type     | Default    |
+| -----------------  | ------------------------ | -------- | ---------- |
+| `enforce_sys_caps` | `BEYLA_ENFORCE_SYS_CAPS` | boolean  | `true`     |
+
+<a id="caps"></a>
+
+When set to `true`, aborts Beyla startup if the required system capabilities
```

Comment:
"If you have set the `enforce_sys_caps` to `true`, if the required system capabilities are not present Beyla aborts its startup and logs a list of the missing capabilities.

If you have set the configuration option to `false`, Beyla logs a list of the missing capabilities only."